README.md

```diff
@@ -40,7 +40,7 @@ If you've found an error in this sample, please file an issue:
 https://github.com/googlesamples/android-UniversalMusicPlayer/issues
 
 Patches are encouraged, and may be submitted by forking this project and
-submitting a pull request through GitHub. Please see CONTRIBUTING.md for more details.
+submitting a pull request through GitHub. Please see [CONTRIBUTING.md](https://github.com/googlesamples/android-UniversalMusicPlayer/blob/master/CONTRIBUTING.md) for more details.
```

Comment:
Would you mind changing this to just "[CONTRIBUTING.md](CONTRIBUTING.md)" so the link will be relative? Thank you!